_posts/network/2023-12-19-create-an-internal-network.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+tags: [Network]
+page_title: Create an internal network
+---
+
+This tutorial guides you through the process of creating an internal network on. 
+An internal network can be used to connect instances together without internet connectivity. 
+If needed, an OpenStack router can be created to allow for internet connectivity and allow for associating floating IPs
+
+---
+
+
+### Creating an internal network
+First we need to create an internal network. For this tutorial we will name it **webserver-network**.  
```

Comment:
I would also change the name of the network since the split version does not need to be webserver-network but can be something else (maybe internal-network?).

Reply:
done